sheets.py

```diff
@@ -44,7 +61,18 @@ def _standardize_field(self, field):
         raise UnsupportedFieldName(field)
 
     def _check_required_cols(self, required_cols, cols, sheet_name):
-        """Validate that all required columns are present."""
+        """
+        Validate that all required columns are present.
+        Then remove the leading '*' from column name
```

Comment:
The asterisk is only removed here to pass the column name to the MissingRequiredFieldError. It's removed elsewhere in the code for processing. I would remove this comment line.